CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-## 2.2.1 (Unreleased)
+## 2.3.0 (Unreleased)
 **Bug Fixes**
 
 **Features**
+- Migrated to the latest [azblob SDK](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azblob).
+- Migrated to the latest [azfile SDK](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/storage/azfile).
+- Migrated from deprecated ADAL to MSAL through the latest [azidentity SDK](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity).
+- Support for cold and premium tier
```

Comment:
We should elaborate this point. Tier change is not supported. We can upload a blob in the access tier specified in the config.
Also update the [base config file](https://github.com/Azure/azure-storage-fuse/blob/main/setup/baseConfig.yaml#L152).